src/app/components/forms/financials/managed-account/managed-account.vue

```diff
@@ -329,7 +339,7 @@ export default class FormFinancialsManagedAccount extends mixins(Wrapper) implem
 		}
 
 		const allPersons = Object.values(this.stripe.persons || []);
-		if (this.account.type === 'individual') {
+		if (this.account.type === this.IndividualAccountType) {
```

Comment:
outside of templates, use the enum directly instead of the one on the class instance. it'll be easier to convert these components in the future then